cmd/ocm-support/patch/subscriptions/cmd.go

```diff
@@ -0,0 +1,101 @@
+package subscriptions
+
+import (
+	"fmt"
+	"io"
+	"os"
+
+	"github.com/openshift-online/ocm-cli/pkg/ocm"
+	v1 "github.com/openshift-online/ocm-sdk-go/accountsmgmt/v1"
+	"github.com/spf13/cobra"
+
+	"github.com/openshift-online/ocm-support-cli/cmd/ocm-support/utils"
+	"github.com/openshift-online/ocm-support-cli/pkg/request"
+	"github.com/openshift-online/ocm-support-cli/pkg/subscription"
+)
+
+var args struct {
+	filter     string
+	dryRun     bool
+	maxRecords int
+}
+
+// CmdPatchSubscriptions represents the subscriptions patch command
+var CmdPatchSubscriptions = &cobra.Command{
+	Use:     "subscriptions [id]",
+	Aliases: utils.Aliases["subscriptions"],
+	Short:   "Patches a Subscriptions for the given ID or subscriptions matching the filter passed",
+	Long:    "Patches a Subscriptions for the given ID or subscriptions matching the filter passed",
+	RunE:    run,
+	Args:    cobra.MaximumNArgs(1),
+}
+
+func init() {
+	flags := CmdPatchSubscriptions.Flags()
+	flags.StringVar(
+		&args.filter,
+		"filter",
+		"",
+		"If non-empty, filters and patches the matching subscriptions.",
+	)
+	flags.BoolVar(
+		&args.dryRun,
+		"dryRun",
+		true,
+		"If false, it will execute the patch command call in instead of a dry run.",
+	)
+	flags.IntVar(
+		&args.maxRecords,
+		"maxRecords",
+		100,
```

Comment:
can we define that in a common place so that the default value will be the same  whenever this flag is used?

Reply:
user utils.MaxRecords